vue/components/info.vue

```diff
@@ -0,0 +1,93 @@
+<template>

```

Comment:
Move this to White's codebase. If it eventually is a common component can be evaluated later